src/pages/TransactionMerge/ConfirmationPage.tsx

```diff
@@ -106,27 +75,17 @@ function ConfirmationPage({route}: ConfirmationPageProps) {
             isASAPSubmitBetaEnabled,
         });
 
-        const reportIDToDismiss = reportID !== CONST.REPORT.UNREPORTED_REPORT_ID ? reportID : targetTransactionThreadReportID;
-        if (reportID !== targetTransaction.reportID && reportIDToDismiss) {
+        const reportIDToDismiss = reportID !== CONST.REPORT.UNREPORTED_REPORT_ID ? reportID : undefined;
```

Comment:
NAB: Should we navigate to "self DM" when user selects "None" report when merging?

Reply:
From the Reports page? no I think it's consistent to stay there.